docs/api-reference/widgets/overview.md

```diff
@@ -10,6 +10,7 @@ This module contains the following widgets:
 
 ### Geospatial Widgets
 - [CompassWidget](./compass-widget.md)
+- [GimbalWidget](./gimbal-widget.md)
```

Comment:
OrbitView isn't really geospatial. How about putting this under "navigation" since that's about widgets related to views